tools/tune/src/src_export_table_2s.m

```diff
@@ -48,6 +48,8 @@
 fprintf(fh, ' *\n');
 fprintf(fh, ' */\n');
 fprintf(fh, '\n');
+fprintf(fh, '/** \\cond GENERATED_BY_TOOLS_TUNE_SRC */\n');
+fprintf(fh, '\n');
```

Comment:
```suggestion
fprintf(fh, '%s\n\n', '/** \cond GENERATED_BY_TOOLS_TUNE_SRC */');
```

Reply:
Yep, I agree

Reply:
> Yep, I agree. I'm not sure what above "Commit suggestion" button will do (create one more commit?) so I will just manually update this PR.

Yes the "commit suggestion" is compatible only with "regular" Github usage: single commit per PR + fixup commits. It's not really compatible with our force-push usage.